qubes_config/global_config.glade

```diff
@@ -1758,7 +1763,131 @@
                           </object>
                           <packing>
                             <property name="left-attach">1</property>
-                            <property name="top-attach">19</property>
+                            <property name="top-attach">20</property>
+                          </packing>
+                        </child>
+                        <child>
+                          <object class="GtkBox">
+                            <property name="visible">True</property>
+                            <property name="can-focus">False</property>
+                            <property name="orientation">vertical</property>
+                            <child>
+                              <object class="GtkLabel">
+                                <property name="visible">True</property>
+                                <property name="can-focus">False</property>
+                                <property name="label" translatable="yes">Preload disposables:</property>
+                                <property name="wrap">True</property>
+                                <property name="xalign">0</property>
+                                <style>
+                                  <class name="main_text"/>
+                                </style>
+                              </object>
+                              <packing>
+                                <property name="expand">False</property>
+                                <property name="fill">True</property>
+                                <property name="position">0</property>
+                              </packing>
+                            </child>
+                            <child>
+                              <object class="GtkBox">
+                                <property name="visible">True</property>
+                                <property name="can-focus">False</property>
+                                <property name="orientation">vertical</property>
+                                <child>
+                                  <object class="GtkLabel">
+                                    <property name="visible">True</property>
+                                    <property name="can-focus">False</property>
+                                    <property name="label" translatable="yes">The maximum number of disposables that can be loaded in the background. This makes using new disposables much faster, but each preloaded disposable will consume memory. Even if you set this number very high, Qubes OS will only ever preload as many disposables as your system can handle, so the actual number of preloaded disposables may be lower. The preloaded disposables will be based on the default disposable template. When this setting is enabled, it takes precedence over the default disposable template's own preloaded disposable setting.</property>
```

Comment:
```suggestion
                                    <property name="label" translatable="yes">Maximum number of disposable qubes (based on default disposable template) that will be created in the background, waiting until they are needed. This improves disposable qube startup time at the cost of increased memory usage. The setting here takes precedence over individual settings of the default disposable template. </property>
```


This is really, really too long. I don't think the details about RAM usage are sensible here? If you think they are, please edit it down to one sentence about resource usage max.